### PR TITLE
Bookings

### PR DESCRIPTION
**Pull request by VladC24**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -19,12 +19,25 @@ class MakersBNB < Sinatra::Base
   end
 
   post '/spaces/new' do
-    Space.create(name: params[:name], description: params[:description], price: params[:price], checkin: params[:checkin], checkout: params[:checkout])
+    Space.create(name: params[:name], description: params[:description],\
```
Comment (breycarr) [inline app.rb]: Would it increase readability to have just one param per line?

File: app.rb [hunk 2]
```diff
@@ -19,12 +19,25 @@ class MakersBNB < Sinatra::Base
...
   end
 
   get '/spaces' do
     @spaces = Space.all
     erb :'spaces/index'
   end
+
+  get '/spaces/bookings/:id' do
+    id = params[:id]
+    @record = Space.first(:id => id)
```
Comment (breycarr) [inline app.rb]: Why did you choose to call this record, as opposed to space?

File: views/spaces/index.erb [hunk 3]
```diff
@@ -1,24 +1,55 @@
...
+  form {
+    text-align: center;
+  }
+
+  div {
+    position: fixed;
+    top: 10;
+    right: 50;
+  }
+
```
Comment (olliebrownlow) [inline views/spaces/index.erb]: We can move all css stuff to the style.css file and keep the views cleaner. To do so, you'll need to add the following line to the top of each view, between the <head> tags: <link rel="stylesheet" href="/style.css"/>

File: spec/helpers/list_space.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+def list_space
```
Comment (breycarr) [inline spec/helpers/list_space.rb]: Use of the helper super helps with readability
Could you replace lines 3 to 7 by using Space.create?

File: views/spaces/bookings.erb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<style>
+</style>
+
+<body>
+  <h2>Please book your space</h2>
+
+  <form action='/spaces/requests' method='post'>
+
+    Hello <%= @record.name %>. <br>
```
Comment (breycarr) [inline views/spaces/bookings.erb]: Would this be clearer as @space.name?

File: views/spaces/index.erb [hunk 6]
```diff
@@ -1,24 +1,55 @@
...
-	h2 {
-		text-align: center;
-		margin-top: 100px;
-	}
-
-	div {
-		text-align: center;
-		margin-top: 50px;
-	}
+
```
Comment (breycarr) [inline views/spaces/index.erb]: I think you could move some of your style information to the style.css with

```
<head>
     <link rel="stylesheet" href="/style.css"/>
  </head>
```

File: views/spaces/index.erb [hunk 7]
```diff
@@ -1,24 +1,55 @@
...
 
+<div>
+  <a href='/spaces'>Spaces</a> | <a href='/spaces/requests'>Requests</a>
+</div>
+
+<body>
+
+<button onclick="location.href='/spaces/new'">List a space</button>
+
+
```
Comment (breycarr) [inline views/spaces/index.erb]: Thanks for adding the list a space button!

File: app.rb (hunk 1 above)
Comment (hughbric) [reply app.rb]: Definitely cleaner.

**Pull request merged**